openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py

```diff
@@ -226,20 +226,20 @@ def create_bulk_transfer(self):
         url = "http://bulk-connector.sandbox.fynarfin.io/batchtransactions"
 
         params = {
-            'type': 'csv',
+            "type": "csv",
         }
         files = {
-            'data': open(csvname, 'rb'),
+            "data": open(csvname, "rb"),
         }
         headers = {
-            'Purpose': 'test payment',
-            'filename': csvname,
-            'X-CorrelationID': 'bd85c0e3-b7bd-40aa-b00f-3240df9d69bd',
-            'Platform-TenantId': os.environ.get("tenantName"),
+            "Purpose": "test payment",
+            "filename": csvname,
+            "X-CorrelationID": "bd85c0e3-b7bd-40aa-b00f-3240df9d69bd",
```

Comment:
Correlation id should be generated for each call, this should not be hardocded.

Reply:
Okay I will pass this as a new UUID each time.